modules/internal_filtering/variables.tf

```diff
@@ -0,0 +1,18 @@
+variable "filtering_default" {
+  description = "Filters as SignalFlow string to enforce the default filtering convention from the template"
+  type        = string
+  default     = ""
+}
+
+variable "filtering_custom" {
+  description = "Filters as SignalFlow string to override the default filtering convention from the user input"
+  type        = string
+  default     = ""
```

Comment:
Maybe it's not a common use-case, but shouldn't be better to set "null" by default, and make your test in output "== null"?

This would allow us to possibily set filtering_custom="" -> no filtering at all (a real empty string is outputed)

Reply:
done!